src/Signin.tsx

```diff
@@ -96,7 +96,12 @@ export class Signin extends PureComponent<Props, State> {
 
   private checkForLogin = async () => {
     try {
-      await fetchLegacyIdentity()
+      if (isFlagEnabled('quartzSession')) {
```

Comment:
do we also want a `CLOUD &&` check here and in the other call?

Reply:
I thought about that and am open to it, but I ended up deciding against it because `fetchIdentity()` already includes a CLOUD check:
![Screen Shot 2022-09-15 at 10 41 20 AM](https://user-images.githubusercontent.com/91283923/190433538-55bb9092-1970-487d-8d04-161ce1d72818.png)

I should remove that comment in `fetchIdentity` (RIP `uiUnificationFlag`)